sql/sql_table.cc

```diff
@@ -5778,7 +5800,14 @@ int mysql_discard_or_import_tablespace(THD *thd,
     DBUG_RETURN(-1);
   }
 
-  error= table_list->table->file->ha_discard_or_import_tablespace(discard);
+  /*
+    ALTER TABLE ... DISCARD/IMPORT TABLESPACE is not supported for tables
```

Comment:
Make sure to add this and other user-visible points to the blueprint so that doc engineering picks them up and documents (i.e. doc will not read through this diff)